src/jit/hwintrinsicxarch.cpp

```diff
@@ -779,11 +787,16 @@ GenTree* Compiler::impSSE2Intrinsic(NamedIntrinsic        intrinsic,
                                     bool                  mustExpand)
 {
     GenTree*  retNode  = nullptr;
-    GenTree*  op1      = nullptr;
-    GenTree*  op2      = nullptr;
     var_types baseType = TYP_UNKNOWN;
+
     switch (intrinsic)
     {
+        case NI_SSE2_SetZeroVector128:
```

Comment:
Importing `NI_SSE2_SetZeroVector128 ` can be table-driven.

Reply:
Will change it